content/webapp/jest.config.js

```diff
@@ -4,5 +4,9 @@ module.exports = {
   setupFiles: ['<rootDir>/jest.setup.js'],
   setupFilesAfterEnv: ['@weco/common/test/setupTests.ts'],
   testEnvironment: 'jest-environment-jsdom',
-  transformIgnorePatterns: ['node_modules(?!/@weco(?!.*node_modules))'],
+  // Transform `uuid` (and scoped @weco packages) because newer `uuid` releases
```

Comment:
The new uuid version uses ESM syntax, so need to tell Jest to transform it. N.B. we only use uuid for Segment tracking I think, so can remove this when we do: https://github.com/wellcomecollection/wellcomecollection.org/issues/12288